apps/dashboard/app/controllers/launchers_controller.rb

```diff
@@ -14,7 +14,8 @@ class LaunchersController < ApplicationController
     :auto_batch_clusters, :auto_batch_clusters_exclude, :auto_batch_clusters_fixed,
     :bc_num_slots, :bc_num_slots_fixed, :bc_num_slots_min, :bc_num_slots_max,
     :bc_num_hours, :bc_num_hours_fixed, :bc_num_hours_min, :bc_num_hours_max,
-    :auto_job_name, :auto_job_name_fixed
+    :auto_job_name, :auto_job_name_fixed,
+    :auto_log_location, :auto_log_location_fixed, :auto_log_location_exclude
```

Comment:
`_exclude` is just for excluding select options. I don't think the form will be passing an `auto_log_location_exclude` list.

Reply:
fixed